megamek/src/megamek/server/Server.java

```diff
@@ -23396,14 +22468,14 @@ private void resolveShutdownCrashes() {
                     if (te.getArmor(hit) <= 0) {
                         break; // hitting IS, not acid-affected armor
                     }
-                    damage = te.getArmor(hit) >= 3 ? 3 : te.getArmor(hit);
+                    damage = Math.min(te.getArmor(hit), 3);
                     r = new Report(6061);
                     r.subject = te_n;
                     r.indent(2);
                     r.add(damage);
                     vDesc.addElement(r);
                 } else if (isPlatoon) {
-                    damage *= 1.5;
+                    damage = (int) Math.round(damage * 1.5);
```

Comment:
I double checked the rules on this. Acid damage to conventional infantry is rounded up.